build/release/README.md

```diff
@@ -0,0 +1,34 @@
+# lnd's Reproducible Build System
+
+This package contains the container that the `lnd` project uses in order to
+build binaries for each new release. As of Go 1.13, with some new build flags,
+binaries are now reproducible, allowing developers to build the binary on
+distinct machines, and end up with a byte-for-byte identical binary.  However,
+this wasn't _fully_ solved in Go 1.13, as the build system still includes the
+directory the binary is built into the binary itself. As a result, our scripts
+utilize a work around needed until Go 1.13.2.  
+
+## Building a New Release
+
+### MacOS/Linux/Windows (WSL)
+
+No prior set up is needed on Linux or MacOS is required in order to build the
+release binaries.  However, on Windows, the only way to build the binaries atm
+is using the Windows Subsystem Linux. You can build the release binaries from
+your `lnd` directory, no matter where it's located.  One can download `lnd` to
```

Comment:
Nit: inconsistent use of "one" and "you" 😛

Reply:
Must've been written at distinct times my two of my distinct selves.